lib/aruba/utils.rb

```diff
@@ -0,0 +1,21 @@
+module Aruba
+  module Utils
```

Comment:
I suggest `Platform` as the name. `Utils` is too general and could become a dumping-ground.